content/en/blog/2022/etcd-integrates-continuous-fuzzing.md

```diff
@@ -0,0 +1,31 @@
+---
+title: etcd Integrates Continuous Fuzzing
+spelling: cSpell:ignore Gyuho
+author:  "[Adam Korczynski](https://twitter.com/AdamKorcz4), [David Korczynski](https://twitter.com/Davkorcz), [Sahdev Zala](https://twitter.com/sp_zala)"
+date: 2022-03-11
+draft: false
+---
+
+In the last few months the team at [Ada Logics](https://adalogics.com) has worked on integrating continuous fuzzing into the etcd project. This was an effort focused on improving the security posture of etcd and ensuring a continued good experience for etcds users. The fuzzing integration involved enrolling etcd in the OSS-Fuzz project and writing a set of fuzzers that would bring the test coverage of etcd up to a mature level. In total, 18 fuzzers were written and 8 bugs were found, demonstrating the value the work has had for etcd both short term and long term. All fuzzers were implemented by way of go-fuzz and when running in OSS-Fuzz instrumented by way of libFuzzer, and as such, etcd uses state-of-the-art open source fuzzing capabilities. 
+The full report of the engagement can be found [here](https://github.com/etcd-io/etcd/blob/main/security/audit_fuzzer_adalogics_2022.pdf). 
```

Comment:
I assume it should also land in `website`, probably: https://github.com/etcd-io/website/tree/main/content/en/community or just blog

Reply:
@ptabor thanks so much for the quick review. It was planned to add as a blog only. The community page seems to focus on community-wide ongoing collaborative events. It has a Twitter handler under `Join the conversation`, and we will tweet blog and report.